src/core/trtserver.h

```diff
@@ -336,6 +336,14 @@ TRTSERVER_EXPORT TRTSERVER_Error* TRTSERVER_ServerModelStatus(
     TRTSERVER_Server* server, TRTSERVER_Protobuf** status,
     const char* model_name);
 
+// Load the requested model.
```

Comment:
Does not return until model is loaded? Returned error indicates if model loaded successfully or not?